test/jdk/java/lang/CompressExpandSanityTest.java

```diff
@@ -25,6 +25,7 @@
  * @test
  * @summary Test compress expand as if the test methods are the implementation methods
  * @key randomness
+ * @run testng/othervm -XX:+UnlockDiagnosticVMOptions -XX:DisableIntrinsic=_expand_i,_expand_l,_compress_i,_compress_l CompressExpandSanityTest
  * @run testng CompressExpandSanityTest
```

Comment:
Can we comment out the annotations so this test is not run by default (i don't know of a better way)

Reply:
@PaulSandoz , commented test tag for sanity test